recipe/meta.yaml

```diff
@@ -43,8 +43,10 @@ outputs:
         - arm-variant * {{ arm_variant_type }}  # [aarch64]
       host:
         - cuda-version {{ cuda_version }}
-        - expat                                 # [aarch64]
+        - expat                                 # [aarch64 or ppc64le]
         - gmp
+        - mpfr                                  # [ppc64le]
```

Comment:
Despite adding `mpfr` here, am seeing [this error on CI]( https://app.travis-ci.com/github/conda-forge/cuda-gdb-feedstock/jobs/604933810#L634 ):

```
  ERROR (cuda-gdb,bin/cuda-gdb): $RPATH/libmpfr.so.4 not found in packages, sysroot(s) nor the missing_dso_whitelist.
```

<br>

As the current version of [`mpfr` in conda-forge is `4.2.0`]( https://github.com/conda-forge/mpfr-feedstock/blob/86b1cebe942849d2f8f40240dbd7a9dc6ab9f8b3/recipe/meta.yaml#L1 ), which contains the `libmpfr.so.6` library, think this is too new for what `cuda-gdb` is looking for as it is looking for `libmpfr.so.4`. This was [last included in `mpfr` version `3.1.6`]( https://abi-laboratory.pro/index.php?view=objects_report&l=mpfr&v1=3.1.6&v2=4.0.0 )